gui/queries/queries.md

```diff
@@ -43,6 +43,10 @@ The queries stored in the query library are also available through the right-han
 (timeframe_selector)=
 ## Selecting a Timeframe
 
+```{note}
+Timeframes are always aligned to one second boundaries. Sub-second timeframes will be automatically truncated to the closest second.
+```
```

Comment:
Should we maybe add this same note to the search.md page where it talks about start/end constraints?
Issue 1296, which is what we are addressing, came up because someone tried to use sub-second precision with start/end. Or not a note on that page, maybe a sentence that calls this out explicitly...? thoughts?

Reply:
Will do